src/test/ui/lint/lint-min-allow-no-effect.stderr

```diff
@@ -0,0 +1,33 @@
+error: #[allow(illegal_floating_point_literal_pattern)] has no effect
+  --> $DIR/lint-min-allow-no-effect.rs:4:10
+   |
+LL | #![allow(illegal_floating_point_literal_pattern)]
+   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+note: lint level defined here
+  --> $DIR/lint-min-allow-no-effect.rs:1:9
+   |
+LL | #![deny(unused_attributes)]
+   |         ^^^^^^^^^^^^^^^^^
+   = note: the minimum lint level for `illegal_floating_point_literal_pattern` is `warn`
+   = note: the lint level cannot be reduced to `allow`
+   = help: remove the #[allow(illegal_floating_point_literal_pattern)] directive
+   = warning: `illegal_floating_point_literal_pattern` was previously accepted by the compiler but is being phased out
+   = warning: it will become a hard error in a future release!
+   = note: for more information, see issue #41620 <https://github.com/rust-lang/rust/issues/41620>
+
+error: #[allow(illegal_floating_point_literal_pattern)] has no effect
```

Comment:
Were you expecting to see this error diagnostic *twice* on this test case?

(It sounds like a bug to me. I know we want this diagnostic to annoy the user, but this might be overkill?)

Reply:
This is more documenting the bug, which I agree that it is... Not sure why it happens... ideas?

Reply:
I don't know. And if you agree that its a bug, then I won't let it block this PR; just make sure we file a bug for it at some point so we can track it.

Reply:
Should I leave a FIXME for now?

Reply:
a FIXME is fine with me